test/cucumber/support/world.js

```diff
@@ -172,6 +172,14 @@ class World {
     return parsed;
   }
 
+  // Hacky coercion function to parse expcected Boolean values
+  // from Gherkin feature suites.
+  toBoolean(string) {
+    if (string === 'true') return true;
+    if (string === 'false') return false;
+    return !!string;
+  }
```

Comment:
I think `JSON.parse('false')` produces false and `JSON.parse('true')` produces true. Perhaps we could use it instead, for a simplicity?

Reply:
How would this work if the `string` is `abc`?
Calling `JSON.parse('abc')` will throw, and if the `JSON.parse(string)` is positioned below the `!!string` it's rendered useless. Maybe I'm not fully understanding what you meant...

Reply:
I wanted to reply with more context, but then I got stuck not being able to find `toBoolean()` used. Where is it being called? I couldn't find `toCamelCase()` or `toPascalCase()` either.